src/bundle/Resources/public/scss/ui/modules/universal-discovery/_selected.locations.scss

```diff
@@ -2,7 +2,7 @@
     background-color: $ibexa-color-white;
     position: fixed;
     top: calculateRem(95px);
-    right: 0;
+    right: calculateRem(16px);
```

Comment:
Added this 1 rem to move this sidebar inside content no idea if this was made properly or just forgotten to update